pkg/apis/policy/v1beta1/clusterimagepolicy_validation_test.go

```diff
@@ -177,6 +177,64 @@ func TestKeyValidation(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:        "Should fail with invalid OCI value",
+			errorString: "invalid value: registry.example.com/repo/*: spec.authorities[0].source[0].oci\ncan only contain the characters `abcdefghijklmnopqrstuvwxyz0123456789_-./`",
+			policy: ClusterImagePolicy{
+				Spec: ClusterImagePolicySpec{
+					Images: []ImagePattern{{Glob: "gcr.io/*"}},
+					Authorities: []Authority{
+						{
+							Key:     &KeyRef{KMS: "kms://key/path"},
+							Sources: []Source{{OCI: "registry.example.com/repo/*"}},
```

Comment:
Check a DockerHub style source without a hostname?

e.g. `mattmoor/signatures` should probably work :)

Reply:
yes, `mattmoor/signatures `will work, but `mattmoor` could be a valid hostname here.